README.md

```diff
@@ -93,9 +93,22 @@ Intercom.registerIdentifiedUser({ userId: 'bob' });
 ```javascript
 Intercom.registerIdentifiedUser({ userId: 'bob' })
 Intercom.updateUser({
-		email: 'email',
-		name: 'name',
-	});
+    // Pre-defined user attributes
+    // (this attributes will be added in custom_attributes as duplicate)
+    email: 'mimi@intercom.com',
+    user_id: 'user_id',
+    name: 'your name',
+    phone: '010-1234-5678',
+    language_override: 'language_override',
+    signed_up_at: 1004,
+    unsubscribed_from_emails: true,
+    companies: [{
+        // Only supported for iOS now
+        // Parameters: IntercomUserAttribtesBuilder.m -> companyForDictionary()
+    }],
+    // Other custom_attributes
```

Comment:
For consistency with how user updating works on iOS and in Intercom, its important that `custom_attributes` is a top level key with an object value. So I suggest this should be:
```
Intercom.updateUser({
    // Pre-defined user attributes
    // (this attributes will be added in custom_attributes as duplicate)
    email: 'mimi@intercom.com',
    user_id: 'user_id',
    name: 'your name',
    phone: '010-1234-5678',
    language_override: 'language_override',
    signed_up_at: 1004,
    unsubscribed_from_emails: true,
    companies: [{
        // Only supported for iOS now
        // Parameters: IntercomUserAttribtesBuilder.m -> companyForDictionary()
    }],
    custom_attributes: {
        my_custom_attribute: 123
    }
    ... 
});
```